packages/components/src/styles/components/dropdown.scss

```diff
@@ -28,6 +28,7 @@ $hds-dropdown-toggle-border-radius: $hds-button-border-radius;
 // TOGGLE/ICON
 
 .hds-dropdown-toggle-icon {
+  position: relative;
```

Comment:
noting here that this element does not contain any custom content (we control all the elements nested within) to there is not risk of introducing layout/positioning issue to our consumers.